docker/build/devdeps.Dockerfile

```diff
@@ -127,8 +128,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends libstdc++-12-de
 ENV BLAS_INSTALL_PREFIX=/usr/local/blas
 ENV OPENSSL_INSTALL_PREFIX=/usr/local/openssl
 ENV OPENSSL_ROOT_DIR="$OPENSSL_INSTALL_PREFIX"
+ENV METIS_INSTALL_PREFIX=/opt/metis
```

Comment:
We want metis in the dev environment for the testing, but not in the release build for the final image. 
One way to achieve this is to follow my suggestion regarding updating the install_prerequisites script, and then edit the two lines
```
install="CMAKE_BUILD_TYPE=Debug/Release FORCE_COMPILE_GPU_COMPONENTS=true"
```
in docker_images.yml to add 
```
install="CMAKE_BUILD_TYPE=Debug/Release FORCE_COMPILE_GPU_COMPONENTS=true METIS_INSTALL_PREFIX="
```

Reply:
updated with my edits